src/custom_gates.jl

```diff
@@ -1,3 +1,35 @@
+"""
+    DoubleExcitation(ϕ)
+
+Generate the matrix representation of the [DoubleExcitation](https://docs.pennylane.ai/en/stable/code/api/pennylane.DoubleExcitation.html) gate.
+
+This gate performs an SO(2) rotation in the subspace {|1100⟩, |0011⟩}, transforming the states as follows:
+
```

Comment:
Can the LaTeX rendering be restored here?

Reply:
Restored!